deep_quoridor/src/agents/alphazero/alphazero.py

```diff
@@ -178,34 +173,22 @@ def end_game(self, env):
         if self.episode_count % self.params.train_every == 0:
             self.train_network()
 
-        wins, losses, ties = 0, 0, 0
-        for reward in self.recent_rewards:
-            if reward == -1:
-                losses += 1
-            elif reward == 0:
-                ties += 1
-            elif reward == 1:
-                wins += 1
-            else:
-                raise ValueError(f"Invalid reward: {reward}")
-
```

Comment:
This code is unused, maybe you needed it for something before, if you want to keep it we can add a comment